oaknational/Oak-Web-Application
# fix: remove margin from subject card [LESQ-625]

## Description

Music year: 1950

- ~remove padding on subject listing cards so full PSED title fits~
- decrease heading size on subject cards to 20px

## How to test

1. Go to https://deploy-preview-2257--oak-web-application.netlify.thenational.academy
2. Should look the same except for reduced font size on subject cards

## Screenshots
<img width=400 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/fdf22bc5-910b-4ed7-ab0d-29efb4bef606">

How it should now look:
<img width=400 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/0d772622-b49d-408a-8f4d-6b67484dac50">


## Checklist
